webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -288,25 +289,27 @@ const NavigationSideBar: React.FC = () => {
                   <p className={css.noWorkspaces}>No pinned workspaces</p>
                 ) : (
                   <ul className={css.pinnedWorkspaces} role="list">
-                    {workspaces.map((workspace) => (
-                      <WorkspaceActionDropdown
-                        key={workspace.id}
-                        trigger={['contextMenu']}
-                        workspace={workspace}>
-                        <li>
-                          <NavigationItem
-                            icon={<DynamicIcon name={workspace.name} size={24} />}
-                            label={workspace.name}
-                            labelRender={
-                              <Typography.Paragraph ellipsis={{ rows: 1, tooltip: true }}>
-                                {workspace.name}
-                              </Typography.Paragraph>
-                            }
-                            path={paths.workspaceDetails(workspace.id)}
-                          />
-                        </li>
-                      </WorkspaceActionDropdown>
-                    ))}
+                    {workspaces
```

Comment:
prob its not from this PR, but spinner position is strange
https://user-images.githubusercontent.com/109113616/207962713-6c2db99c-2e74-4b94-bb65-6b1f3ed7a3a2.mov

Reply:
I agree when I was working on pinned workspaces I thought it should be indented a bit more, but I don't know what style options we have then